docs/core/porting/windows-compat-pack.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Porting to .NET Core - Using the Windows Compatibility Pack
+description: Learn about the Windows Compatibility Pack and how can you use it to port existing .NET Framework code to .NET Core
+keywords: .NET, .NET Core, Windows, Compatibility
+author: terrajobst
+ms.author: mairaw
+ms.date: 11/13/2017
+ms.topic: article
+ms.prod: .net-core
+---
+
+# Using the Windows Compatibility Pack
+
+One of the most common issues that developers face when porting their existing
+code to .NET Core is that they depend on APIs and technologies that only exist
+in the .NET Framework. The *Windows Compatibility Pack* is about providing many
+of these technologies so that building .NET Core applications as well as .NET
+Standard libraries becomes much more viable for existing code.
+
+This package is a logical [extension of .NET Standard 2.0](../whats-new/index.md#api-changes-and-library-support)
+that significantly increases API set and existing code compiles with almost no
+modifications. But in order to keep the promise of .NET Standard ("it is the set
+of APIs that all .NET implementations provide"), this didn't include
+technologies that can't work across all platforms, such as registry, Windows
+Management Instrumentation (WMI), or reflection emit APIs.
```

Comment:
technical question: can the APIs that are included in the pack work across all platforms?

Reply:
Some of them. I wrote below:

> It provides about 20,000 APIs, including Windows-only as well as cross-platform
> APIs from the following technology areas:

Reply:
I'll talk to you later about this one. Let's leave as-is for now. Given that the pack includes some that are not cross-platform there must be other reasons they were not included, no?